chatsky/core/message.py

```diff
@@ -257,6 +257,32 @@ class MediaGroup(Attachment):
     chatsky_attachment_type: Literal["media_group"] = "media_group"
 
 
+class Origin(BaseModel):
+    original_message: Optional[Any] = None
+    origin_interface: Optional[str] = None
```

Comment:
I don't like repeating origin in the fields. Just `message` and` interface` should be better.